plots/charts/index.html

```diff
@@ -16,14 +16,10 @@
 </head>
 
 <body>
-  <nav>
+  <nav id="nav">
     View results:
-    <a href="./index.html">grouped by metric</a>
-    <a href="./metrics-per-site.html">grouped by site</a>
-    <a>bar charts per site</a>
   </nav>
-  <script src="./out/generatedResults.js"></script>
-  <script src="./bars-per-site.js"></script>
+  <script src="../generated-results.js"></script>
```

Comment:
IMO if you're going to leave gen-results in the root of `out` then you should put `index.html` there as well.